# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../seamonkey/no-logging.patch
# Copyright (C) 2015 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

From: Chris Double <chris.double@double.co.nz>
Bug 1128161 - Fix build error when logging is disabled - r=ajones


diff --git a/dom/media/mediasource/MediaSourceUtils.cpp b/dom/media/mediasource/MediaSourceUtils.cpp
index 44a40f5..83bba60 100644
--- a/mozilla/dom/media/mediasource/MediaSourceUtils.cpp
+++ b/mozilla/dom/media/mediasource/MediaSourceUtils.cpp
@@ -6,17 +6,16 @@
 #include "MediaSourceUtils.h"
 
 #include "prlog.h"
 #include "mozilla/dom/TimeRanges.h"
 #include "nsPrintfCString.h"
 
 namespace mozilla {
 
-#if defined(PR_LOGGING)
 nsCString
 DumpTimeRanges(dom::TimeRanges* aRanges)
 {
   nsCString dump;
 
   dump = "[";
 
   for (uint32_t i = 0; i < aRanges->Length(); ++i) {
@@ -26,11 +25,10 @@ DumpTimeRanges(dom::TimeRanges* aRanges)
     ErrorResult dummy;
     dump += nsPrintfCString("(%f, %f)", aRanges->Start(i, dummy), aRanges->End(i, dummy));
   }
 
   dump += "]";
 
   return dump;
 }
-#endif
 
 } // namespace mozilla
